### PR TITLE
Improve messages in the agent installation

### PR DESCRIPTION
**Pull request by yenienserrano**
**Description:**

Changes were made to the messages in the agent installation guide.

- From `You can predefine the Wazuh server address with the enrollment.dns setting` to `This is the address the agent uses to communicate with the Wazuh server. It can be an IP address or a fully qualified domain name (FQDN).`

- From `Running this command on a host with an agent already installed upgrades the agent package without enrolling the agent. To enroll it, see the Wazuh documentation` to `If the installer finds another Wazuh agent in the system, it will upgrade it preserving the configuration.`

**Test:**

1. Navigate to `Agents`
2. Click on `Deploy a new agent`
3. Look at the changes

**Issues:**

Improve messages on agent installation walkthrough [#4037](https://github.com/wazuh/wazuh-kibana-app/issues/4037)
Improve messages in the agent installation instructions [#4036](https://github.com/wazuh/wazuh-kibana-app/issues/4036)

**Screenshot:**
image
image

### CODE REVIEW

File: public/controllers/agent/components/register-agent.js [hunk 1]
```diff
@@ -515,8 +515,7 @@ export const RegisterAgent = withErrorBoundary(
                   color="warning"
                   title={
                     <>
-                      Running this command on a host with an agent already installed upgrades the
-                      agent package without enrolling the agent. To enroll it, see the{' '}
+                      Running this command will not enroll the agent if it is doing an upgrade. See{' '}
                       <EuiLink href="https://documentation.wazuh.com/current/user-manual/registering/index.html">
```
Comment (AlexRuiz7) [inline public/controllers/agent/components/register-agent.js:519]: There is a missign requirement on #4036 

> Also, the link to the documentation should open a new tab in the browser.

Add `target="_blank"` to this link, as seen [here](https://codesandbox.io/s/fcgt4y?file=/demo.js:198-213).

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]

**Review by AlexRuiz7 — APPROVED** [review]
LGTM!

**Comment by yenienserrano** [comment]
Updated proposal
image

**Comment by yenienserrano** [comment]
Updated proposal
image

**Review by Mayons — APPROVED** [review]

**Comment by yenienserrano** [comment]
Updated issue
- #4036

image

**Pull request merged**